.ostree/get_ostree_data.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
```

Comment:
I assume this file should be consistent as how it is used in other roles.

Reply:
Yes.  It will be managed by https://github.com/linux-system-roles/.github